### DISCUSSION TITLE
what all llm's does this project support currently except openai

### DISCUSSION THREAD

**Discussion by Greatz08** [Skyvern-AI/skyvern#90] [General]
:-)

**Comment by andr0s** [comment]
yeah I'd also like to know. From the docker config file, I see openai, azure, anthropic. Any chance to support a local LLM, Groq, or any other platform? OpenAI costs can be scary tbh

**Comment by suchintan** [comment]
Hi! Skyvern currently supports GPT-4V, GPT-4O, Claude-3 Sonnet, Claude-3 Opus. It will also likely work with Gemini series of models!
